show/main.py

```diff
@@ -290,6 +475,16 @@ def summary():
     # Clean up
     os.remove(PLATFORM_TEMPLATE_FILE)
 
+#
+# 'hardware' command
+#
+
+@cli.command()
+def hardware():
+     """Show local hardware information"""
+     comm="sudo dmidecode -t 1,2,10"
```

Comment:
Change to `command = "..."` for consistency ("comm" -> "command" and add spaces around equals sign).